stactools_landsat/stactools/landsat/utils.py

```diff
@@ -119,10 +121,14 @@ def transform_stac_to_stac(item: Item,
                             crs = opened_asset.crs.to_epsg()
                             # Check to ensure that all information is present
                             if not shape or not transform or not crs:
-                                raise STACError(f"Failed setting shape, transform and csr from {asset.href}")
+                                raise STACError(
+                                    f"Failed setting shape, transform and csr from {asset.href}"
+                                )
 
                     except RasterioIOError as io_error:
-                        raise STACError(f"Failed loading geotiff, so not handling proj fields, {io_error}")
+                        raise STACError(
+                            f"Failed loading geotiff, so not handling proj fields, {io_error}"
+                        )
 
                 item.ext.projection.set_transform(transform, asset=asset)
                 item.ext.projection.set_shape(shape, asset=asset)
```

Comment:
```suggestion
                    item.ext.projection.set_transform(transform, asset=asset)
                    item.ext.projection.set_shape(shape, asset=asset)
```

This was missed on the last go-round.

Reply:
Good Morning @gadomski,
     It wasn`t missed this part of the code is right. As I explained to you, ```item.ext.projection.set_transform``` and ```item.ext.projection.set_transform``` don't set anything in the Item, instead, it sets in the asset and because of that they MUST be inside of the loop.